index.html

```diff
@@ -136,6 +136,25 @@ <h3 class="aboutmesub">Skills</h3>
                 </ul>
             </div>
         </section>
+        <section class="forms">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in </h2>
+            <form action="https://formspree.io/f/form_id" method="post">
```

Comment:
- [ ] Your not using a valid form spree id, kindly sign up to the [formspree service](https://formspree.io/html) and create an id to use here so you can receive the emails sent from your portfolio